### PR TITLE
[NEW BAR SIGN] Le Café Silencieux

### PR DESCRIPTION
**Pull request by DATA-xPUNGED**
## About The Pull Request

Adds my entry to the barsign contest, Le Café Silencieux (The Silent Café, in french)
TheCafeSilentieux
## Why It's Good For The Game

Variety good!
## Changelog
:cl:DATA_
sprite: A new barsign is now available! Invoke your inner mime in this french-themed café: Le Café Silencieux!
/:cl:

### CODE REVIEW

**Comment by DATA-xPUNGED** [comment]
Now we wait for the conflicts...

File: code/game/machinery/barsigns.dm [hunk 1]
```diff
@@ -292,6 +292,12 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/barsign, 32)
 	desc = "Honk."
 	neon_color = "#FF998A"
 
+/datum/barsign/le_cafe_silencieux
+	name = "Le Café Silencieux"
+	icon = "le_cafe_silencieux"
```
Comment (Thunder12345) [inline code/game/machinery/barsigns.dm:297]: `icon` ⟶ `icon_state`

Comment (Kapu1178) [reply code/game/machinery/barsigns.dm:297]: did the author even test this

Comment (DATA-xPUNGED) [reply code/game/machinery/barsigns.dm:297]: wait, so why exactly are linters failing?

Comment (Thunder12345) [reply code/game/machinery/barsigns.dm:297]: > incorrect, this is a datum, not an actual obj. /datum/barsign uses `icon` for this, poor naming.

Correct, this is a datum, not an actual obj. /datum/barsign uses `icon_state` for this, good naming

Comment (Thunder12345) [reply code/game/machinery/barsigns.dm:297]: > wait, so why exactly are linters failing?

You’re using the wrong var name, pull the latest master to your branch then update your code to use the right name

Comment (DATA-xPUNGED) [reply code/game/machinery/barsigns.dm:297]: Oh there we go, i have to solve this VSC stuff later though, i can't seem to fetch upstream from it at all

**Comment by Wallemations** [comment]
Honestly it feels a little bit slow, though I suppose making it too fast would just make it seem like it's blinking rather than slow-fading.

Maybe try removing either 1 or 0.5 from each frame delay, see which one you prefer better?

**Comment by DATA-xPUNGED** [comment]
> Honestly it feels a little bit slow, though I suppose making it too fast would just make it seem like it's blinking rather than slow-fading.
> 
> Maybe try removing either 1 or 0.5 from each frame delay, see which one you prefer better?

I specifically made it to be slow, , it's supposed to invoke a feeling of a refined place

**Comment by Wallemations** [comment]
yknow what sure, it's good

**Pull request merged**